build.properties

```diff
@@ -1,4 +1,4 @@
 minecraft_version=1.12.2
-forge_version=14.23.0.2528
+forge_version=14.23.0.2532
 mod_version=7.0.1
-mappings_version=snapshot_nodoc_20171102
+mappings_version=snapshot_nodoc_20171108
```

Comment:
There isn't any reason to constantly update these to the latest bleeding-edge versions.
Every time either of the Forge or MCP versions change forces everyone to have to reset their dev workspace with setupDecompWorkspace, and the latter always runs the risk of having to continually update the source to match new MCP names.